include/arch/arc/arch.h

```diff
@@ -105,7 +105,7 @@ extern "C" {
 #define Z_ARCH_THREAD_STACK_MEMBER(sym, size) \
 	struct _k_thread_stack_element \
 		__aligned(POW2_CEIL(STACK_SIZE_ALIGN(size))) \
-		sym[POW2_CEIL(size) + \
+		sym[POW2_CEIL(STACK_SIZE_ALIGN(size)) + \
```

Comment:
We do not do the same for ARM, as far as I can see.

Reply:
the min MPU region size on ARM is 32 bytes. on ARC it's 2K.

Reply:
Alright. I leave this to ARC-guys, anyways..